### PR TITLE
Fixed notification style for DevHub post

### PR DESCRIPTION
**Pull request by HarisSQA**
Fixed notification issue with DevHub.

### CODE REVIEW

File: src/NearOrg/Notifications/Notification.jsx [hunk 1]
```diff
@@ -124,14 +124,48 @@ const Left = styled.div`
...
+      .filter(([_key, nullable]) => (nullable ?? null) !== null)
+      .map(([key, value]) => `${key}=${value}`)
+      .join("&");
+
+  return `/${nearDevGovGigsWidgetsAccountId}/widget/gigs-board.pages.${widgetName}${
+      linkPropsQuery ? "?" : ""
+  }${linkPropsQuery}`;
+}
+
+console.log({props})
```
Comment (shelegdmitriy) [inline src/NearOrg/Notifications/Notification.jsx:167]: Remove it please

Comment (HarisSQA) [reply src/NearOrg/Notifications/Notification.jsx:167]: Sure! Sorry about that, didn't see it 😀

Comment (shelegdmitriy) [reply src/NearOrg/Notifications/Notification.jsx:167]: Your code looks good! Just need to check that locally. May I ask you to add a couple of screenshots here?

**Comment by HarisSQA** [comment]
Hi @shelegdmitriy @calebjacob ,

I I can merge the PR if everything looks good and it’s approved. Thanks!

**Review by calebjacob — APPROVED** [review]

**Comment by charleslavon** [comment]
Nice work @HarisSQA We can start with this enhancement and consider additional customizations for devhub later  cc @ravigautham

**Pull request merged**